rulesets.go

```diff
@@ -203,6 +203,7 @@ type RulesetRuleActionParametersHTTPHeader struct {
 
 type RulesetRuleActionParametersOverrides struct {
 	Enabled    bool                                    `json:"enabled,omitempty"`
```

Comment:
from your issue in cloudflare/terraform-provider-cloudflare#1249, you want this to be a boolean pointer and it will always send if it is set explicitly.

```suggestion
	Enabled    *bool                                    `json:"enabled,omitempty"`
```